.pre-commit-config.yaml

```diff
@@ -2,11 +2,12 @@
 # (pre-commit, environment.yml, requirements.txt, pyproject.toml, ...)
 
 repos:
-- repo: https://github.com/jsh9/pydoclint
-  rev: 0.2.1
+- repo: https://github.com/pycqa/flake8
+  rev: 7.0.0
   hooks:
-    - id: pydoclint
-      args: [--config=pydoclint.toml]
+    - id: flake8
```

Comment:
Just to be on the safe side: are you sure that flake8 is not executed itself in this way? I'm just asking because flake8 is already covered through ruff, so we should avoid duplicate runs.

Reply:
Technically, this executes flake8 but only checks for all "DOC" errors, which are exactly the ones that are checked by pydoclint.